src/containers/Versions/Versions.tsx

```diff
@@ -55,67 +64,70 @@ export const Versions = ({nodes = [], versionToColor}: VersionsProps) => {
             </div>
         );
     };
-    const renderGroupedNodes = () => {
-        const tenantNodes = getGroupedTenantNodes(nodes, versionToColor, groupByValue);
-        const storageNodes = getGroupedStorageNodes(nodes, versionToColor);
-        const otherNodes = getOtherNodes(nodes, versionToColor);
-        const storageNodesContent = storageNodes?.length ? (
-            <>
-                <h3>Storage nodes</h3>
-                {storageNodes.map(({title, nodes: itemNodes, items, versionColor}, index) => (
+
+    if (loading && !wasLoaded) {
```

Comment:
Here I removed `renderGroupedNodes` function and added loader. GH considers its as a new code :(